test/integration/disruption/disruption_test.go

```diff
@@ -513,6 +517,25 @@ func waitToObservePods(t *testing.T, podInformer cache.SharedIndexInformer, podN
 	}
 }
 
+// createPDBUsingRemovedAPI creates a PDB directly using etcd.  This is must *ONLY* be used for checks of compatibility
+// with removed data. Do not use this just because you don't want to update your test to use v1.  Only use this
+// when it actually matters.
+func createPDBUsingRemovedAPI(ctx context.Context, etcdClient *clientv3.Client, etcdStoragePrefix, nsName string, betaPDB *v1beta1.PodDisruptionBudget) error {
+	betaPDB.APIVersion = v1beta1.SchemeGroupVersion.Group + "/" + v1beta1.SchemeGroupVersion.Version
+	betaPDB.Kind = "PodDisruptionBudget"
+	betaPDB.Namespace = nsName
+	betaPDB.Generation = 1
```

Comment:
also need to set creationTimestamp (to time.Now()) and uid (to a random uuid) to mimic what the API server does for you:

https://github.com/kubernetes/kubernetes/blob/8633adbb0702d277a225df6c5703f0b63b9638f3/staging/src/k8s.io/apiserver/pkg/registry/rest/meta.go#L35C1-L39